test/ng/compileSpec.js

```diff
@@ -3155,6 +3155,55 @@ describe('$compile', function() {
       });
     });
 
+    it('should denormalise interpolation symbols in templates correctly, if different to the current symbols', function() {
+      angular.module('symbol-test', [])
+        .directive('myDirective', function() {
+          return {
+            template: '<span foo=\'{"ctx":{"id":3}}\'>[[1 + 2]]</span>'
+          };
+        })
+        .config(function($compileProvider) {
+          $compileProvider.moduleSymbols('symbol-test', '[[', ']]');
+        });
+
+      module('symbol-test');
+
+      inject(function($compile) {
+        element = $compile('<div><div my-directive></div></div>')($rootScope);
+        expect(element.children('div').children('span').attr('foo')).toBe('{"ctx":{"id":3\\}\\}');
+        expect(element.text()).toEqual('{{1 + 2}}');
+        $rootScope.$apply();
+        expect(element.text()).toEqual('3');
```

Comment:
What about `.attr('foo')` ? I believe it is still escaped and I think this is not what we want...